configs/openeo-cloud.json

```diff
@@ -166,7 +170,7 @@
   },
   "scrape_start_urls": false,
   "js_render": true,
-  "js_wait": 1,
+  "js_wait": 2,
```

Comment:
I don't understand. For example https://docs.openeo.cloud/processes/ is not included, but that's a single page with basically no links that is not in the index yet. It's directly mentioned as an entry page and a side map would only contain this specific page, too, right?

Reply:
If there's client-side rendering, it's fine to have `js_render` and `js_wait` option, but for missing pages/link discovery it's optimal to have a `sitemap` added to the config (you can also have both the sitemap and client-side rendering options)

Reply:
My assumption is that they are missing due to slow rendering, because the URL is actually listed specifically in one of the configs as start URL, but it doesn't shop up in searches.

Reply:
Make sense then!

Reply:
Looking forward to the new infra, because we have 99% SSR, but only 1% of very important pages being slow JS pages and increasing js_wait until it works for all pages seems rather excessive.

Reply:
Indeed, I'll make sure to include your configs in the next batch :)

Reply:
Great, thanks. Let me know once it's done and I'll happily update the configs to reflect that.